authors/apps/articles/models.py

```diff
@@ -37,3 +39,16 @@ def slug_generator(sender, instance, *args, **kwargs):
 
 
 pre_save.connect(slug_generator, sender=Article)
+
+
+class Like_Dislike(models.Model):
```

Comment:
```suggestion
class LikeDislike(models.Model):
```

Reply:
Yes @3Nakajugo, you are right. Python conventions for naming classes should follow CamelCase conventions. However, that being a model, I would not want to delete migrations due to the name change. I am going to look out for this in the future and follow your recommendations. Thank you, Edna.